### PR TITLE
Tick the scheduler when in client mode as well.

### PR DESCRIPTION
**Pull request by Dockter**
Signed-off-by: Dockter <dockter@almuramc.com>

### CODE REVIEW

**Comment by kashike** [comment]
~qa

**Comment by Dockter** [comment]
Added check so the client isn't double ticking the queue.

File: src/main/java/org/spongepowered/mod/SpongeMod.java [hunk 1]
```diff
@@ -276,8 +276,10 @@ public void onTick(TickEvent.ServerTickEvent event) {
     @SideOnly(Side.CLIENT)
     @SubscribeEvent
     public void onTick(TickEvent.ClientTickEvent event) {
-        if (event.phase == TickEvent.Phase.START) {
-            this.scheduler.tickSyncScheduler();
+        if (!net.minecraft.client.Minecraft.getMinecraft().isIntegratedServerRunning()) {
```
Comment (Zidane) [inline src/main/java/org/spongepowered/mod/SpongeMod.java]: Fully qualified import is un-needed as import lines are just for our benefit. When the code is compiled, everything is made fully qualified.

Comment (Dockter) [reply src/main/java/org/spongepowered/mod/SpongeMod.java]: TIL, ty.

**Comment by Zidane** [comment]
Manually merged.

**Comment by simon816** [comment]
I'm not sure about this... `ClientTickEvent` occurs on the client thread, running the scheduler on any thread other than the server thread will cause issues.

I'm not sure why you want to be running server tasks outside of the server lifecycle. This seems like something you'd want from a client API, operating on a separate client-side scheduler

**Comment by Zidane** [comment]
@simon816 Hasn't caused any issues in the two weeks this has been live. I'm well aware that a Client scheduler should be added but it isn't a priority to do that refactor currently.

**Pull request closed without merging**